source/Octopus.Tentacle.Client/Scripts/ScriptExecutionOrchestrator.cs

```diff
@@ -24,6 +24,7 @@ internal class ScriptExecutionOrchestrator : IDisposable
         readonly Action<ScriptStatusResponseV2> onScriptStatusResponseReceived;
         readonly Func<CancellationToken, Task> onScriptCompleted;
         readonly TimeSpan onCancellationAbandonCompleteScriptAfter;
+        readonly RpcRetrySettings settings;
```

Comment:
Nit: make the property names give more context of what the property is e.g. retrySettings

Reply:
ah yeah that's fair. `settings` seemed OK before, as it was more generic to the TentacleClient, but a more specific name makes sense now.